subprojects/emcee/queue-api-models/src/main/kotlin/com/avito/emcee/queue/Bucket.kt

```diff
@@ -8,4 +8,6 @@ public data class Bucket(
     val payloadContainer: PayloadContainer,
     val analyticsConfiguration: Any,
     val workerCapabilityRequirements: List<String>
-)
+) {
+    public companion object
```

Comment:
Tell me could we don't use that hack?

Reply:
We use it throughout the project. We can use factory functions or something else, but I don't see anything wrong with exposing the companion objects.

Reply:
What if we make it `internal`